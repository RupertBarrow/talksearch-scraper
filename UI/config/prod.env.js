'use strict'
module.exports = {
  NODE_ENV: '"production"',
  API_URL: '"https://algolia-talksearch.herokuapp.com"'
}
