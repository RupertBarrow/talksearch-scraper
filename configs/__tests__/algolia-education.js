import config from '../algolia-education.js';
import helper from '../config-helper.js';

describe('Algolia Education', () => {
  describe('transformData', () => {
    let current;
    beforeEach(() => {
      current = input => config.transformData(input, helper);
    });

    it('Algolia Build 101 - Push Data - for Javascript developers', () => {
      const input = {
        video: {
          title: 'Algolia Build 101 - Push Data - for Javascript developers',
        },
      };

      const actual = current(input);

      expect(actual).toMatchSnapshot();
    });
  });
});
