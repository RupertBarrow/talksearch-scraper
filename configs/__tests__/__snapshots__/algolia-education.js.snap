// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Algolia Education transformData Algolia Build 101 - Push Data - for Javascript developers 1`] = `
Object {
  "language": "Javascript",
  "speakers": Array [],
  "video": Object {
    "title": "Push Data - for Javascript developers",
  },
}
`;
